SparkPost/2web2ui
# FE-1111 - Sentry now renders a tag for isHibanaEnabled

[FE-1111](https://jira.int.messagesystems.com/browse/FE-1111)

### What Changed
- Added new tag to Sentry that passes on the user's `optons.ui.isHibanaEnabled` state

### How To Test
1. Sign in to the app and make sure Hibana is disabled
1. Go to `/error-boundary` and take a look at the network request made to Sentry - verify that `tags: { isHibanaEnabled: false }` is in the request body
1. Enable Hibana
1.Go to `/error-boundary` and verify the request body now includes `tags: { isHibanaEnabled: true }`
1. Log out and create a new account without the feature flag enabled
1. Go to `/error-boundary` and verify the request body contains `tags: { isHibanaEnabled: false }`

### To Do
